django_project/changes/views/changelog_github.py

```diff
@@ -280,14 +285,23 @@ def download_all_referenced_images(request, **kwargs):
                                 if not block:
                                     break
                                 handle.write(block)
-                            img_url = file_path.replace('/home/web', '')
+
+                            # remove MEDIA_ROOT to obtain image_file relative
+                            # path and add the MEDIA_URL path into the file
+                            # path e.g img_url: /media/images/entries/img-1.png
+                            img_url = file_path.replace(
+                                settings.MEDIA_ROOT,
+                                re.sub(r'/$', '', settings.MEDIA_URL))
                             html = html.replace(image, img_url)
                             html = re.sub(r"alt=\".*?\"", "", html)
+                        # Take the first image set in the pull request
+                        # and set it as default for the entry and remove
+                        # it from the body
                         if i == 0:
-                            # Take the first image set in the pull request
-                            # and set it as default for the entry and remove
-                            # it from the body
-                            entry.image_file = img_url.replace('/media', '')
+                            # remove MEDIA_URL and assign the value to
+                            # image_file
+                            entry.image_file = img_url.replace(
+                                settings.MEDIA_URL, '')
```

Comment:
It doesn't make sense because the right side is url but the left side is image field, CMIIW?
Why not use the `file_path` ?
Or better way, use the file stream as input to `image_file.save` like in this article: https://nikhilhaas.com/downloading-and-saving-image-to-imagefield-in-django/

Check how they uses `cStringIO` to stream the download, then pass it as `ContentFile` to image_field's `save`.